cmd/mixtool/main.go

```diff
@@ -48,15 +50,28 @@ func main() {
 
 // If no jPath is given, we check if ./vendor exists in the current directory and use it.
 // filename should be the path to the mixin root file (ie 'mixin.libsonnet').
-func availableVendor(filename string, jPathsFlag []string) []string {
+func availableVendor(filename string, jPathsFlag []string) ([]string, error) {
 	if len(jPathsFlag) == 0 {
-		// Look for a vendor dir along side the root mixin file.
-		basedir := path.Dir(filename)
-		vendor := path.Join(basedir, "vendor")
-		_, err := os.Stat(vendor)
+		workdir, err := filepath.Abs(filename)
+		if err != nil {
+			return nil, err
+		}
+
+		workdir = filepath.Dir(workdir)
+		root, err := jpath.FindParentFile("jsonnetfile.json", workdir, "/")
```

Comment:
I wanted to avoid that as I references "tkrc.yaml", which isn't very applicable to mixtool.  https://github.com/grafana/tanka/blob/master/pkg/jsonnet/jpath/jpath.go#L74

Reply:
It would be unused and unnoticed in normal operations. 

Doesn't quite matter if we skip this for now